templates/Gemfile.erb

```diff
@@ -56,3 +55,7 @@ end
 group :staging, :production do
   gem "rack-timeout"
 end
+
+group :staging do
```

Comment:
I think it should also be in development and test. You don't want to be sending emails there either.